sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/Framework/AuthenticationEventDataTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using Xunit;
+using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Framework;
+using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.TokenIssuanceStart.Data;
+
+namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Tests.Framework
+{
+    /// <summary>
+    /// Tests for AuthenticationEventData
+    /// </summary>
+    public class AuthenticationEventDataTests
+    {      
+        public static string TenantId = "00000000-0000-0000-0000-000000000001";
+        public static string AuthenticationEventListenerId = "00000000-0000-0000-0000-000000000002";
+        public static string CustomAuthenticationExtensionId = "00000000-0000-0000-0000-000000000003";
+
+        public static string BuildDataString(
```

Comment:
You could use the AuthenticationEventJsonElement to build the JSON rather than the string. You could dynamically add properties.

Reply:
For now, I want to be less reliant on helper functions; particularly AuthenticationEventJsonElement as it a wrapper for System.Text.Json. Pretty sure we'll be doing a hard shift to a new version or completely adopting it.